js/ui/grid_core/ui.grid_core.filter_row.js

```diff
@@ -601,10 +600,9 @@ var ColumnHeadersViewFilterRowExtender = (function() {
 })();
 
 var DataControllerFilterRowExtender = {
-    _calculateAdditionalFilter: function() {
-        var that = this,
-            filters = [that.callBase()],
-            columns = that._columnsController.getVisibleColumns();
+    _calculateFilterRowAdditionalFilter: function() {
```

Comment:
Return logic from _calculateFilterRowAdditionalFilter to _calculateAdditionalFilter